File: Solutions/58. Length of Last Word/58. Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        s+=' ';
        string tmp="";
        for(int i=0;i<s.size();++i){
            if(tmp.size() and s[i]!=' ')
                tmp="";
            while(s[i]!=' ')
                tmp+=s[i++];
        }
        return tmp.size()?tmp.size():0;
    }
};

File: Solutions/58. Length of Last Word/58. Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s+=' '
        tmp,i="",0
        while i<len(s):
            if len(tmp)>0 and s[i]!=' ':
                print(tmp)
                tmp=""
            while s[i]!=' ':
                tmp+=s[i]
                i+=1
            i+=1
        return len(tmp) if len(tmp)>0 else 0
                